.github/workflows/build-build-tools-image.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       matrix:
         arch: [ x64, arm64 ]
 
-    runs-on: ${{ fromJson(format('["self-hosted", "dev", "{0}"]', matrix.arch)) }}
+    runs-on: ${{ fromJson(format('["self-hosted", "large", "{0}"]', matrix.arch)) }}
```

Comment:
We don't run this workflow if there are no changes in `Dockerfile.build-tools`; let's add some no-op changes there (or update some packages we have) to trigger this workflow.

Reply:
tested (added one \r at the end of dockerfile). Built with success.